script.js

```diff
@@ -31,51 +36,44 @@ function addEvents() {
     });
   });
 }
+
 // populate dom with the list
 const addBtn = document.querySelector('#add-btn');
 function showBooks(list) {
-  const booksListDiv = document.querySelector('#books-container');
-  list.forEach((lists, i) => {
+  const booksListDiv = document.querySelector('#List');
+  // let colorChange = 0;
+  for (let i = 0; i < list.length; i += 1) {
     const div = document.createElement('div');
     const pTitle = document.createElement('p');
-    pTitle.innerHTML = list[i].title;
+    pTitle.innerHTML = `"${list[i].title}" by ${list[i].author}`;
     div.appendChild(pTitle);
-    const pAuthor = document.createElement('p');
-    pAuthor.innerHTML = list[i].author;
-    div.appendChild(pAuthor);
     const btn = document.createElement('button');
     btn.className = 'remove';
     btn.innerHTML = 'Remove';
     btn.id = list[i].id;
     div.appendChild(btn);
-    const hr = document.createElement('hr');
-    div.appendChild(hr);
+    // const hr = document.createElement('hr');
+    // div.appendChild(hr);
```

Comment:
- [ ] Kindly please, remove this introduced old piece of commented-out code.  That way, the project will be cleaner 💪